CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
-[cygnus-ngsi-ld] Creation of the new cygnus-ngsi-ld bundle for receiving NGSI-LD notifications, the first sink provided is PostgreSQL (#1875)
+[cygnus-ngsi][Generic Aggregation] Add name mappings flag to all sinks who uses generic aggregation.
```

Comment:
Is there any issue number associated to the fix?

Reply:
Not sure about it. I thing there isn't.

Reply:
Ok. NTC.